### ISSUE TITLE
event-stream@3.3.6 is a dependency

### ISSUE THREAD

**Issue by nmccready** [nonoroazoro/vscode-syncing#47]
https://blog.npmjs.org/post/180565383195/details-about-the-event-stream-incident

Get event-stream@3.3.6 out of your repo please.

```
❯ yarn why event-stream
yarn why v1.12.3
[1/4] 🤔  Why do we have the module "event-stream"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "event-stream@3.3.6"
info Reasons this module exists
   - "vscode#gulp-symdest" depends on it
   - Hoisted from "vscode#gulp-symdest#event-stream"
   - Hoisted from "vscode#gulp-vinyl-zip#event-stream"
   - Hoisted from "vscode#gulp-remote-src-vscode#event-stream"
   - Hoisted from "vscode#gulp-untar#event-stream"
✨  Done in 0.56s
```

```
❯ yarn
yarn install v1.12.3
[1/5] 🔍  Validating package.json...
warning syncing@2.0.2: The engine "vscode" appears to be invalid.
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
error https://registry.yarnpkg.com/flatmap-stream/-/flatmap-stream-0.1.1.tgz: Extracting tar content of undefined failed, the file appears to be corrupt: "Unexpected end of data"
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this comma
```

dominictarr/event-stream#116

**Comment by nonoroazoro** [comment]
@nmccready  I'll remove it asap, thanks.

**Comment by nonoroazoro** [comment]
@nmccready  Fixed in [latest release (v2.0.5)](https://github.com/nonoroazoro/vscode-syncing/releases/latest).

**Comment by nonoroazoro** [comment]
I'd close this issue if there's no other problems :)
